HLTrigger/Configuration/python/HLTrigger_EventContent_cff.py

```diff
@@ -15,7 +16,16 @@
         'keep FEDRawDataCollection_source_*_*',
         'keep GlobalObjectMapRecord_hltGtStage2ObjectMap_*_*',
         'keep edmTriggerResults_*_*_*',
-        'keep triggerTriggerEvent_*_*_*'
+        'keep triggerTriggerEvent_*_*_*',
+        'keep *_hltFEDSelectorL1_*_*',
+        'keep *_hltScoutingCaloPacker_*_*',
+        'keep *_hltScoutingEgammaPacker_*_*',
+        'keep *_hltScoutingMuonPackerCalo_*_*',
+        'keep *_hltScoutingMuonPacker_*_*',
+        'keep *_hltScoutingPFPacker_*_*',
+        'keep *_hltScoutingPrimaryVertexPackerCaloMuon_*_*',
+        'keep *_hltScoutingPrimaryVertexPacker_*_*',
+        'keep edmTriggerResults_*_*_*'
```

Comment:
hi @fwyzard - i guess we don't need this keep statement twice?

Reply:
if you mean just
```python
        'keep edmTriggerResults_*_*_*' 
```
no, we don't.

It is there because I've merged the keep statements from two different output modules, and this was the easiest way to do it.

I can try to find a better way...